gemini/evaluation/metrics_based_gemini_evaluation.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<p>Add a link to where that table is from. I'd recommend using a markdown table instead of an image. Also, add links to definitions of these metrics.</p><p> </p><p>Formatting <code>## Overview</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/482/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>